pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -40,6 +43,18 @@ const (
 	referenceCloser   = ')'
 )
 
+var kedaNamespace, _ = util.GetClusterObjectNamespace()
+
+// getWatchNamespace returns the namespace the operator should be watching for changes
+func getRestrictSecretAccess() bool {
+	const RestrictSecretAccessEnvVar = "RESTRICT_SECRET_ACCESS"
```

Comment:
For consistency, maybe we should use `KEDA_RESTRICT_SECRET_ACCESS`

Reply:
Agreed, updated